### ISSUE TITLE
Gojs console issue

### ISSUE THREAD

**Issue by alpeshkalena** [NorthwoodsSoftware/GoJS#118]
![image](https://user-images.githubusercontent.com/41985261/100914486-265c0d00-34f9-11eb-8540-a6a28cac2ea3.png)

Hi, I am using go.js with angular and i am getting error which is described in attached screenshot.
project is successfully compiling but getting error in chrome browser's console.
Can you please guide me or provide any solution or any solutions links for error solution?

Thanks in advance

**Comment by simonsarris** [comment]
Is your code trying to minify go.js further? That may be an issue.

Otherwise, if you can offer some code so that we can reproduce this, we can take a look.

**Comment by alpeshkalena** [comment]
Hi,

**Comment by alpeshkalena** [comment]
Hi, we have remove optimization but still getting the same error. I am sending you error from js which is shown in below video link.
https://jmp.sh/4QecSt9 -- This is video link so you can actual idea what i am getting.

**Comment by awulkan** [comment]
Hey, we're also getting this problem when upgrading from version 2.1.25 to 2.1.30.
It's a big Vue 2 application, so I don't think I can reproduce it easily.
When inspecting the code, it seems like it transpiles into this:

```js
var Ff, Hf = new pt, Yf = "object" === typeof g.document && g.document.currentScript ? g.document.currentScript.src : "", zf = Yf, Uf = ["Layout", "GoJSScriptURL"], Vf = this;
Uf[0]in Vf || !Vf.execScript || Vf.execScript("var " + Uf[0]);
```
Where I think `Vf` is undefined.

The temporary solution for me was to downgrade to 2.1.25.

**Comment by simonsarris** [comment]
Is it possible for either of you to make me a reproducible sample of what you're seeing?

**Comment by WalterNorthwoods** [comment]
@awulkan, that code is not in the `go.js` library.

Are you sure that you do not have any code that is using a minified name?  GoJS defines no two letter property or method names except for **InputEvent.up**.

**Comment by simonsarris** [comment]
We have reproduced this issue and will have a fix soon.

**Comment by awulkan** [comment]
@WalterNorthwoods No, this is code that's gone through build process, just like for @alpeshkalena. So the names are minified.

@simonsarris Great! Sorry for not creating a reproducible example in time.

**Comment by simonsarris** [comment]
Hello, please update to the just-released version `2.1.31`, which should fix these issues.

Sorry for the trouble. Let me know if you encounter anything else.

**Comment by awulkan** [comment]
I can verify that the fix solved my problem at least. Thanks.
